pkg/reconciler/instances/istio/action.go

```diff
@@ -9,11 +19,18 @@ import (
 )
 
 const (
-	istioNamespace = "istio-system"
-	istioChart     = "istio-configuration"
+	istioNamespace        = "istio-system"
+	istioChart            = "istio-configuration"
+	istioImagePrefix      = "eu.gcr.io/kyma-project/external/istio/proxyv2"
+	retriesCount          = 5
+	delayBetweenRetries   = 10
+	sleepAfterPodDeletion = 10
 )
 
 type ReconcileAction struct {
+	commander       istioctl.Commander
+	istioProxyReset proxy.IstioProxyReset
+	performer       actions.IstioPerformer
 }
 
 func (a *ReconcileAction) Run(version, profile string, config map[string]interface{}, context *service.ActionContext) error {
```

Comment:
This function became very long, it could be separated into a few shorter functions.